src/components/PersonForTable.tsx

```diff
@@ -0,0 +1,68 @@
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../types';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+export const PersonForTable: React.FC<Props> = ({ person, people }) => {
+  const { personSlug } = useParams();
+  const selectedPerson = personSlug ? personSlug : '';
+  function findParentLink(name: string | null) {
+    if (name === null) {
+      return '-';
+    }
+    const parent = people.find(person => person.name === name);
+
+    return parent?.slug;
```

Comment:
The `findParentLink` function should return `null` or `undefined` instead of `parent?.slug` when the parent is not found. This will prevent generating incorrect links when the parent does not exist.